php-practice/section2.php

```diff
@@ -117,6 +139,18 @@ function printNewline() {
 // HINT: Use a foreach loop
 // =======================================================================
 
+$sumArray = array(1, 2, 3, 4); 
+
+function ($sumArray) {
+	foreach ($sumArray as $value);
+	$total = 0;
+	$total += $value;
+	return ($total);
```

Comment:
So the foreach syntax is messed up here.

 It has to have curly braces, and the code inside of the curly braces is what it does  each time that it loops over an element.

So like this:

```
$total = 0;

foreach ($arr as $value) {
  // ADD STUFF TO TOTAL HERE
}

return $total;
```